src/wallet/rpcwallet.cpp

```diff
@@ -3402,15 +3407,15 @@ UniValue z_gettotalbalance(const UniValue& params, bool fHelp)
         throw runtime_error(
             "z_gettotalbalance ( minconf includeWatchonly )\n"
             "\nReturn the total value of funds stored in the node’s wallet.\n"
-            "\nCAUTION: If the wallet contains watch-only zaddrs, the returned private balance may be larger than the actual balance,"
-            "\nbecause spends cannot be detected with incoming viewing keys.\n"
+            "\nCAUTION: If the wallet contains watch-only Sprout shielded addresses, the returned private balance may"
+            "\nbe larger than the actual balance, because spends cannot be detected with incoming viewing keys.\n"
             "\nArguments:\n"
             "1. minconf          (numeric, optional, default=1) Only include private and transparent transactions confirmed at least this many times.\n"
             "2. includeWatchonly (bool, optional, default=false) Also include balance in watchonly addresses (see 'importaddress' and 'z_importviewingkey')\n"
             "\nResult:\n"
             "{\n"
             "  \"transparent\": xxxxx,     (numeric) the total balance of transparent funds\n"
-            "  \"private\": xxxxx,         (numeric) the total balance of private funds\n"
+            "  \"private\": xxxxx,         (numeric) the total balance of private funds (in both Sprout and Sapling addresses)\n"
```

Comment:
This raises the question of whether there is supposed to be a difference between "private" and "shielded" (there isn't).

Reply:
I don't quite see the context (shielded isn't mentioned in this line). Please open an issue detailing the help text rework you want (which likely needs to happen elsewhere too in order to be consistent).

Reply:
Filed https://github.com/zcash/zcash/issues/3486